src/MigrationTools/Configuration/Processing/WorkItemMigrationConfig.cs

```diff
@@ -35,6 +35,7 @@ public class WorkItemMigrationConfig : IWorkItemProcessorConfig
         public int AttachmentMaxSize { get; set; }
         public bool CollapseRevisions { get; set; }
         public bool LinkMigrationSaveEachAsAdded { get; set; }
+        public bool GenerateMigrationComment { get; set; }
```

Comment:
Can you also add a default value to the constructor with `GenerateMigrationComment = true;`?